### PR TITLE
Update link to NPM package guidelines in readme

### PR DESCRIPTION
**Pull request by rgaban**
- Issues #[508](https://github.com/LaunchPadLab/lp-components/issues/508): NPM package guidelines link has moved 
- Updated link to npm package guidelines link in readme

## Author Checklist

- [ ] Add unit test(s)
- [x] Update version in package.json (see the [versioning guidelines](https://github.com/LaunchPadLab/opex-public/blob/master/gists/npm-package-guidelines.md#pull-requests-and-deployments))
- [x] Update documentation (if necessary)
- [ ] Add story to storybook (if necessary)
- [x] Assign dev reviewer

### CODE REVIEW

**Review by chawes13 — COMMENTED** [review]
Thank you! Can you bump the version in `package.json` as a patch release?

https://semver.org/#summary

**Comment by chawes13** [comment]
@rgaban I have exciting news! We were able to change the default branch to `main`, so the appropriate link is now: https://github.com/LaunchPadLab/opex-public/blob/main/gists/npm-package-guidelines.md

**Review by chawes13 — CHANGES_REQUESTED** [review]
Can you bump the version and swap out `master` for `main`? 🙏

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Author Checklist
 
 - [ ] Add unit test(s)
-- [ ] Update version in package.json (see the [versioning guidelines](https://github.com/LaunchPadLab/opex-public/blob/master/gists/npm-package-guidelines.md#pull-requests-and-deployments))
+- [ ] Update version in package.json (see the [versioning guidelines](https://github.com/LaunchPadLab/opex-public/blob/main/gists/npm-package-guidelines.md#pull-requests-and-deployments))
```
Comment (rgaban) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: @chawes13 found that the pull request request template needed to be updated as well!

**Review by chawes13 — APPROVED** [review]
🎉

**Pull request merged**